src/ui/index.js

```diff
@@ -86,32 +86,42 @@ function init () {
     const versionText = `Version ${version}`
     document.getElementById('menuVersion').innerHTML = versionText
 
-    setTimeout(function () {
+    let initialMenuHidingTimeoutID = setTimeout(function () {
+      initialMenuHidingTimeoutID = null;
```

Comment:
The only issue I see before merging is the use of semicolons here and there.
Sorry but as per our CONTRIBUTING.md , we use [standardjs style](https://standardjs.com/)
(we will likely add some automated linting like in the IO package very soon)

Reply:
fixed.